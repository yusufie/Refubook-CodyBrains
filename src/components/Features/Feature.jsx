import React from 'react';
import star from '../../assets/star.svg';

const Feature = ({ item }) => {
  return (
    <div className=" max-w-sm flex flex-col items-center   gap-4 py-5">
      <div className="flex items-center justify-start w-full gap-4">
        <img src={star} alt="" />
        <h4 className="text-lg text-refubookLightBlack text-left font-bold">
          {item?.title}
        </h4>
      </div>
      <p className="text-md text-refubookLightBlack font-medium text-left">
        {item?.text}
      </p>
    </div>
  );
};

export default Feature;
