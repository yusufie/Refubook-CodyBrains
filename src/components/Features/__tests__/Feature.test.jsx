import renderer from 'react-test-renderer';
import Feature from '../Feature';

describe('Feature', () => {
  it('renders correctly with props passed', () => {
    const tree = renderer
      .create(<Feature key={1} title="Test Title" text="Test Text" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
