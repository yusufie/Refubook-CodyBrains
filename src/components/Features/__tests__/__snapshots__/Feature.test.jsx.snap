// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Feature renders correctly with props passed 1`] = `
<div
  className=" max-w-sm flex flex-col items-center   gap-4 py-5"
>
  <div
    className="flex items-center justify-start w-full gap-4"
  >
    <img
      alt=""
      src="star.svg"
    />
    <h4
      className="text-lg text-refubookLightBlack text-left font-bold"
    />
  </div>
  <p
    className="text-md text-refubookLightBlack font-medium text-left"
  />
</div>
`;
